content/docs/learning-guide/tutorials/entities-and-prefabs/spawn-a-prefab.md

```diff
@@ -0,0 +1,136 @@
+
+The script you've created thus far spawns a prefab instance at the spawner entity's location when the spawner entity is activated. You can add the script to the spawner entity's Script Canvas node, and press **CTRL-G** to test the script. In the final steps of the tutorial you'll learn how to despawn a prefab.
+
+## Delay and despawn
+
+In this section, you'll add a three second delay to the script, after which, the prefab despawns.
+
+1. Add a **Delay** node from the **Timing** group to the graph canvas and connect it with the following steps:
+
+    1. Connect the **On Spawn Completed** pin of the Spawn node to the **Start** pin of the Delay node.
+
+    1. On the Delay node, set the **Start Time** property to `3` to create a three second delay.
+
+1. Add a **Despawn** node from the **Prefab/Spawning** group to the graph canvas and connect it with the following steps:
+
+    1. Connect the **Done** pin of the Delay node to the **Request Despawn** pin of the Despawn node.
+
+    1. On the Despawn node, **right-click** on the **SpawnTicket** pin and select **Convert to Reference** from the context menu. The property is automatically set to the **SpawnTicket** variable you created in step **2**. You can choose the {{< icon settings.svg>}} settings icon in the **SpawnTicket** field to select a variable reference if necessary.
+
+1. Press **CTRL-S** to save the script. The following image displays the complete Script Canvas graph.
+
+    {{< image-width "/images/learning-guide/tutorials/entities-and-prefabs/final-script-canvas.png" "900" "The final script canvas graph.">}}
+
+## Test the results
+
+Now you'll add the script to the spawner entity and test the script.
+
+1. In O3DE Editor, select the spawner entity.
+
+1. In Entity Inspector, In the Script Canvas component, choose the {{< icon browse-edit-select-files.svg >}} file browser icon and select the `spawn-despawn-prefab.scriptcanvas` file you created.
+
+    ![Set the script canvas file property](/images/learning-guide/tutorials/entities-and-prefabs/set-script-canvas-source.png)
+
+1. Press **CTRL+G** to run the level and test the script.
+
+    {{< video src="/images/learning-guide/tutorials/entities-and-prefabs/spawn-despawn-finished.mp4" info="Click and drag to connect Script Canvas nodes." autoplay="true" loop="true" >}}
+
+## Conclusion and next steps
+
+Congratulations! You've learned to spawn and despawn a prefab instance. These are critical concepts to working in O3DE. Here are some ideas for building on what you've learned in this tutorial:
+
+* Can you randomize the delay before despawning the prefab instance?
+* See if you can work out how to spawn the prefab instance at a location of your choosing. Hint: Play with the transform properties on the **Spawn** Script Canvas node.
+* Can you spawn multiple prefab instances at different locations from a single spawner entity?
```

Comment:
The only thing that is not covered here is how to delegate SpawnTicketInstance's to other ScriptCanvas'es via ScriptEvents. This can be a super useful functionality when we want spawned entities to control their own lifespan.

Reply:
We'll follow up with an advanced tutorial and include this.